src/loaders.py

```diff
@@ -149,17 +124,19 @@ def get_datagen(self, X, y, train_mode, loader_params):
         if train_mode:
             dataset = self.dataset(X, y,
                                    train_mode=True,
-                                   image_augment=self.image_augment,
-                                   image_augment_with_target=self.image_augment_with_target,
+                                   image_augment=self.image_augment_train,
+                                   image_augment_with_target=self.image_augment_with_target_train,
                                    mask_transform=self.mask_transform,
-                                   image_transform=self.image_transform)
+                                   image_transform=self.image_transform,
+                                   image_source=self.dataset_params.image_source)
         else:
             dataset = self.dataset(X, y,
                                    train_mode=False,
-                                   image_augment=None,
-                                   image_augment_with_target=None,
+                                   image_augment=self.image_augment_inference,
```

Comment:
@apyskir mhm exactly how I intended to use it. It seems to me that this option lets one do pretty much any loading acrobations (tta, random aug, prep). Would you agree?